### PR TITLE
Update power-bi-automate-visual.md

### PR DESCRIPTION
**Pull request by sbasi19**
Updates to documentation for the Power Automate custom visual GA

### CODE REVIEW

**Comment by sbasi19** [comment]
There is a typo the line "To select your preferred environment for creating your flow, use the environment picker in the Power Automate portal." should say "To select your preferred environment for creating your flow, use the environment picker in the Power Automate custom visual."

File: powerbi-docs/create-reports/power-bi-automate-visual.md [hunk 1]
```diff
@@ -44,10 +45,15 @@ When you create a Power Automate visual in a Power BI report, your end-users can
 
 2. Scroll, resize the visual, or select the **Focus mode** icon to see all the instructions.
 
-    :::image type="content" source="media/power-bi-automate-visual/automate-new-visual.png" alt-text="Power Automate new visual added to the report.":::
+    ![image](https://github.com/MicrosoftDocs/powerbi-docs/assets/79485235/7886eb16-d331-485b-b07c-f8d088e497b9)
 
 3. After you’ve reviewed the instructions, resize the button and place it where you’d like on the report.
 
+## Change the environment in which your flow is created
+To select your preferred environment for creating your flow, use the environment picker in the Power Automate portal. This includes environments where you have any built-in security role, as well as any environments where you are a co-owner of one or more flows. If you cannot find your preferred environment, [click here](https://learn.microsoft.com/en-us/power-platform/admin/troubleshoot-missing-environments) to learn more about the requirements for accessing environments in product experiences.
```
Comment (sbasi19) [inline powerbi-docs/create-reports/power-bi-automate-visual.md:53]: Power Automate portal should say Power Automate custom visual

**Comment by Jak-MS** [comment]
@maggiesMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerbi-docs/create-reports/power-bi-automate-visual.md (hunk 1 above)
Comment (maggiesMSFT) [reply powerbi-docs/create-reports/power-bi-automate-visual.md:53]: got it. Updated in https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/11527

**Comment by maggiesMSFT** [comment]
closed, replaced with https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/11527

**Comment by maggiesMSFT** [comment]
#please-close

**Comment by maggiesMSFT** [comment]
closed, replaced with https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/11527

**Pull request closed without merging**